src/components/pages/debug/DebugPage.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+function generateError() {
+  throw new Error('Generated error from Debug page')
+}
+
+const Debug = () => (
+  <main className="debug-main">
+    <button
+      className="db-create-error"
```

Comment:
```suggestion
      className="db-generate-error"
```
Pour être raccord avec la fonction du domain.